src/sentry/consumers/__init__.py

```diff
@@ -186,14 +186,7 @@ def ingest_events_options() -> list[click.Option]:
     ),
 ]
 
-
-_INGEST_SPANS_OPTIONS = multiprocessing_options(default_max_batch_size=100) + [
```

Comment:
just to clarify, did u remove this because it's not being used anywhere, unrelated to the control plane work?

Reply:
yes, it's dead code so i thought i'd clean it up